curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65ca0e8eb2c9c215269d6a66.md

```diff
@@ -1,15 +1,13 @@
 ---
-id: 655a536e99be288210f01451
-title: Step 44
+id: 65ca0e8eb2c9c215269d6a66
+title: Step 54
 challengeType: 20
-dashedName: step-44
+dashedName: step-54
 ---
 
 # --description--
 
-The `inorder_traversal` method is responsible for performing an in-order traversal of the binary search tree. It returns the keys of the nodes in sorted order.
-
-In-order traversal is a depth-first binary tree traversal algorithm that visits the left subtree, the current node, and then the right subtree.
+The `inorder_traversal` method is responsible for performing an in-order traversal of the binary search tree. It returns the keys of the nodes in sorted order. It would use the helper method `_inorder_traversal` to achieve this.
```

Comment:
```suggestion
The `inorder_traversal` method is responsible for performing an in-order traversal of the binary search tree. It returns the keys of the nodes in sorted order. It will use the helper method `_inorder_traversal` to achieve this.
```